bot/persistence/queries.py

```diff
@@ -32,6 +32,14 @@
 IS_REACTION_ROLE_UNIQUE = "SELECT EXISTS(SELECT 1 FROM ReactionRoleUniquenessGroup WHERE MessageID = ?)"
 
 
+# Translations
+GET_TRANSLATION = "SELECT Translation FROM Translation WHERE MessageID = ? AND FlagEmoji = ? LIMIT 1"
```

Comment:
Is it good to have the same name for table and column? Maybe rename column to TranslatedMessage or sth like that